hpe_3par_kubernetes_manager.py

```diff
@@ -310,12 +352,39 @@ def hpe_list_secret_objects_names(namespace):
         raise e
 
 
+def hpe_list_service_objects(namespace):
+    try:
+        service_list = k8s_core_v1.list_namespaced_service(namespace=namespace)
+        return service_list
+    except client.rest.ApiException as e:
+        #print("Exception :: %s" % e)
+        logging.getLogger().error("Exception :: %s" % e)
+        raise e
+
+
+def hpe_list_service_objects_names(namespace):
+    try:
+        service_names = []
+        service_list = hpe_list_service_objects(namespace=namespace)
+        for service in service_list.items:

```

Comment:
try this

service_names = [service.metadata.name for service in service_list.items]